cookbooks/software/redis/templates/default/redis.conf.erb

```diff
@@ -1,11 +1,13 @@
-daemonize no
+daemonize yes
```

Comment:
If you remove `Type=forking` from the systemd service file, can we prevent redis from daemonizing? It feels more robust to let systemd run redis in foreground rather than have redis go to background and hoping that systemd keeps track of it. Or am I missing something?

Reply:
Good point! I'll try that!